STYLEGUIDE.md

```diff
@@ -46,17 +46,19 @@ import (
 	"context"
 	"fmt"
       // above are standard library packages
+	appsv1 "k8s.io/api/apps/v1"
 	"k8s.io/client-go/tools/record"
 	"sigs.k8s.io/kustomize/k8sdeps/kunstruct"
 	"sigs.k8s.io/kustomize/pkg/target"
 	ktypes "sigs.k8s.io/kustomize/pkg/types"
       // above are k8s library packages
 	"github.com/kudobuilder/kudo/pkg/util/health"
 	"github.com/kudobuilder/kudo/pkg/util/template"
-	appsv1 "k8s.io/api/apps/v1"
```

Comment:
This is actually the right sorting according to `goimports`.

Reply:
Not with the flags we pass it from `make imports`.

Reply:
Oh sorry, you're right!